documentation/privilege_escalation.md.erb

```diff
@@ -0,0 +1,74 @@
+# Escalating privilege with Bolt
+
+By default Bolt executes on remote systems as the same user it connects to the system with.
+Sometimes connection and execution need to be done as separate users, for example needing to install
+a package as the root user on a system that doesn't allow incoming connections as the root user.
+Bolt has several configuration options for setting which user to execute as, how to escalate to
+that user, and how to run commands as that user.
+
+> **Note:** Executing as a different user is only supported when using the Bash shell over the SSH
+> or Local transport.
+
+### Limitations
+
+The specified `run-as` user will only be used if:
+
+* The target shell is bash
+* The transport is Local or SSH
+* The configured `run-as` user is different than the connecting `user`
+
+## Configuring `run-as`
+
+### Command-line options
+
+<% @run_as_options.each do |option| -%>
+
+#### `<%= option[:long] -%><%= option[:arg] %>`
+
+<%= option[:desc] %>
+<% end %>
+
+### Configuration file options
+
+<% @run_as[:options].each do |option, data| -%>
+#### `<%= option %>`
+
+<%= data[:desc] %>
+
+<% if data[:type] -%>
+- **Type:** <%= data[:type] == TrueClass ? 'Boolean' : data[:type] %>
+<% end -%>
+<% end %>
+
+## How `run-as` works
+
+For all execution Bolt builds a command string to be run on the remote
+system. The command is built from many configuration options including
+interpreters, task input method, and `run-as` configuration. When `run-as` is
+set the following is prepended to the command being run:
```

Comment:
```suggestion
Each time Bolt executes, it builds a command string to run on the remote
system. The command is built from many configuration options, including
interpreters, task input methods, and the `run-as` configuration. When 
`run-as` is set, the following is prepended to the command:
```
Not sure about method/s. Is it methods, the method?

Reply:
I think method, since it's one option?